lib/Backends/NNPI/Importer.cpp

```diff
@@ -1694,18 +1599,15 @@ class ChannelwiseQuantizedConvolutionNodeImporter : public INNPINodeImporter {
             nodeValueName(glowChannelwiseQuantizedConv->getInput()),
             nodeValueName(glowChannelwiseQuantizedConv->getFilter()),
             nodeValueName(glowChannelwiseQuantizedConv->getBias()),
-            symlowpInputName,
-            symlowpOutputName,
         },
         {
             nodeValueName(glowChannelwiseQuantizedConv->getResult()),
-            symlowpInputName,
-            symlowpOutputName,
         });
 
     return nnpiNetworkAddConvolutionOp(
         importer.getNetwork(), glowChannelwiseQuantizedConv->getName().begin(),
-        symlowpInputName.c_str(), symlowpOutputName.c_str(),
+        nodeValueName(glowChannelwiseQuantizedConv->getInput()).c_str(),
```

Comment:
This was the change we didn't include from 5.2.5, because it broke A-step, right? Has it been fixed? I noticed that we blacklisted the RWQ-FC test on A-step but not this one. In case we run into issues when integrating CC: @arunm-git @hl475

Reply:
@jfix71 We have disabled the failing for A-step (added to block list).  
See lib/Backends/NNPI/tests/NNPIOperatorTest.cpp:140 
{"rowwiseQuantizedFCTestSymmetric/0", TestBlacklist::A0AnyEngine},